plugin/trino-google-sheets/src/main/java/io/trino/plugin/google/sheets/SheetsSplit.java

```diff
@@ -33,35 +33,17 @@
 {
     private static final int INSTANCE_SIZE = toIntExact(ClassLayout.parseClass(SheetsSplit.class).instanceSize());
 
-    private final String schemaName;
```

Comment:
>Alternatively, we could keep a name property and fill in the sheetId instead.

The semantics of this field should not depend on the context.

You can add a dedicated implementation of `ConnectorSplit` which would have the `id` field. Or add another field here, and make them all optional. 
i'm leaning towards the latter, because the semantics of the Split is the same in both cases, and they only differ in the available metadata.

Reply:
fixed.